Cargo.toml

```diff
@@ -103,7 +103,7 @@ exclude = ["tools/"]
 resolver = "2"
 
 [workspace.package]
-version = "0.1.0"
+version = "0.2.3-dev"
```

Comment:
Reading up on SemVer, this seems like the correct way to do it: `-dev` effectively represents a pre-release version, whereas `2.2+` would indicate a release with annotated *empty* build metadata (which is what `+` represents).